include/caf/ble_adv.rst

```diff
@@ -0,0 +1,138 @@
+.. _caf_ble_adv:
+
+CAF: Bluetooth LE advertising module
+####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+When enabled for an application, the |ble_adv| is responsible for controlling the Bluetooth LE advertising for Bluetooth LE Peripheral device.
+
+This module can only work together with the :ref:`CAF Bluetooth LE state module <caf_ble_state>`.
+The Bluetooth LE state module is a core Bluetooth module in the :ref:`lib_caf` (CAF).
+
+Configuration
+*************
+
+To enable the |ble_adv|, complete the following steps:
+
+1. Enable and configure the :ref:`CAF Bluetooth LE state module <caf_ble_state>`.
+#. Define the Bluetooth device name using the :option:`CONFIG_BT_DEVICE_NAME` Kconfig option.
+#. Define the Bluetooth device appearance using the :option:`CONFIG_BT_DEVICE_APPEARANCE` Kconfig option.
+#. Set the :option:`CONFIG_CAF_BLE_ADV` Kconfig option.
```

Comment:
We need an additional step about creating the configuration file to avoid confusion with the next step.

Reply:
Ok I will add additional description here